Makefile

```diff
@@ -23,41 +28,61 @@ install : all
 	$(INSTALL) $(installfiles) $(DESTDIR)$(installdir)
 
 clean :
-	$(RM) *.0 *.signing_policy *.pem.txt
+	$(RM) *.0 *.signing_policy *.pem
```

Comment:
This change is fine, but note that this `make clean` rule relies on using `.signing_policy.txt` suffixes for the sources so it only deletes the `*.signing_policy` derived files, not the `*.signing_policy.txt` sources.

Reply:
Ah, I missed this detail in my rush! This should be good to go now